controller/space.go

```diff
@@ -411,6 +439,8 @@ func ConvertSpaceFromModel(ctx context.Context, db application.DB, request *goa.
 	relatedOwnerByLink := rest.AbsoluteURL(request, fmt.Sprintf("%s/%s", identitiesEndpoint, sp.OwnerId.String()))
 	relatedCollaboratorList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/collaborators", spaceIDStr))
 	relatedFilterList := rest.AbsoluteURL(request, "/api/filters")
+	selfSpaceTemplateURL := rest.AbsoluteURL(request, app.SpaceTemplateHref(sp.SpaceTemplateID))
+	relatedSpaceTemplateURL := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/spacetemplate", spaceIDStr))
```

Comment:
> are we using related anywhere ?

yes in here https://github.com/almighty/almighty-core/pull/1148/files/77a1a69af135534abaf20263b8f01e31a5ac8e6b#diff-539b5d9c6c13722feba14308fd61a6cdR511